foundation-extensions/places/README.md

```diff
@@ -1,13 +1,179 @@
-# Adobe Experience Platform Places Service
+# Adobe Experience Platform Location Service
 
-Adobe Experience Platform Places Service provides an SDK extension which allows you to act based on the location of your users. This extension is the interface to the [Experience Platform Places Service Web Services APIs](https://experienceleague.adobe.com/docs/places/using/web-service-api/places-web-services.html?lang=en). 
+Adobe Experience Platform Location Service provides an SDK extension which allows you to act based on the location of your users. This extension is the interface to the [Location Service Web Services APIs](https://experienceleague.adobe.com/docs/places/using/web-service-api/places-web-services.html?lang=en).
 
 The SDK extension listens for events that contain GPS coordinates and geofence region events, and dispatches new events that are processed by the Rules Engine. The SDK extension also retrieves and delivers a list of the nearest POI for the app data that retrieves from the APIs. The regions returned by the APIs are stored in cache and persistence, which allows limited offline processing.
 
-To get started with implementing Adobe Experience Platform Places Service on your mobile app, review documentation on [Places extension](https://experienceleague.adobe.com/docs/places/using/places-ext-aep-sdks/places-extension/places-extension.html).
+`Places` is the mobile SDK supporting the Location Service.
 
-## Reference
+## Add the AEP Places extension to your app
```

Comment:
Before getting to configuring Places SDK in app
Should we write about 
`Configuring Places extension in Data Collection UI` ?

Reply:
added